content/en/apps/guides/data/analytics/testing-dbt-models.md

```diff
@@ -0,0 +1,138 @@
+Add custom data tests for aggregation models to ensure the data is as expected (Only specific models need these tests)
+Add unit tests for complex logic e.g. data math, window functions e.t.c. (not required but good to have especially if model is covered by data tests)
+
+Our project is organized with a `/models` directory containing SQL files and YAML files for generic tests, and a `/test` directory with folders for fixtures and singular tests.
+
+```
+./
+    /models
+        /contacts
+            /tests
+                contacts.yml  --> unit tests
+                persons.yml
+            contact.sql
+            contacts.yml      --> generic data tests
+            patient.sql
+            person.sql
+            place.sql
+        /forms
+        /meta
+        /root
+        /users
+    /test
+        /fixtures                                     --> input data for unit tests
+            data_record/
+                data_record_initial_expected.csv
+                data_record_source_table_initial.csv
+                data_record_document_metadata_initial.csv
+        /sqltest                                       --> singular data tests
+            contact.sql
+            data_record.sql
+            patient.sql
+```
+
+### Data Tests
+
+***Generic data tests*** are included in the model's YAML file under the `data-tests` tag, ensuring basic validations like `relationships`, `not_null`, and `unique`. The following image contains an extract of the properties of the  `contacts.yml` file.
+{{< figure src="contacts-yml.png" link="contacts-yml.png" class=" center col-16 col-lg-12" >}}
+
+***Singular data tests*** are located in the `/test/sqltest/` folder and check data synchronization between source and final tables. For example, the following image shows the test to ensure that data between the `source` table and `contact` tables is synchronized correctly.
+Records that should be deleted are properly removed from both tables.
+All relevant fields between the two tables match, preserving data accuracy and integrity.
+{{< figure src="contact-sql.png" link="contact-sql.png" class=" center col-16 col-lg-12" >}}
+
+### Unit Tests
+
+Unit tests are defined in a YML file inside a `tests` folder in each `model` group folder. The input format of choice is CSV using fixtures, defined in `/test/fixture` folder.
+
+The following image shows the content in `/models/contacts/tests/contacts.yml` file showing the test to validate transformation and data integrity for `contact` model.
+{{< figure src="contact-unit-test.png" link="contact-unit-test.png" class=" center col-16 col-lg-12" >}}
+
+The following images shows the content of the input fixtures.
+{{< figure src="contact-document-metadata-initial.png" link="contact-document-metadata-initial.png" class=" center col-16 col-lg-12" >}}{{< figure src="contact-source-table-initial.png" link="contact-source-table-initial.png" class=" center col-16 col-lg-12" >}}
```

Comment:
Please replace the image with a code block. This will increase readability, make the page consistent with the rest of the docs site, and support version control changes easier.